native_transport_ssl_test.py

```diff
@@ -40,7 +40,7 @@ def test_connect_to_ssl(self):
         except NoHostAvailable:
             pass
 
-        if cluster.version() >= '4.0':
+        if cluster.version() >= '4.0' and cluster.version() < '5.0':
```

Comment:
If NotSslRecordException is being thrown we should assert that happens in 5.0

Reply:
there is "else" block